src/tests/ftest/pool/destroy_rebuild.py

```diff
@@ -28,34 +29,36 @@ def test_destroy_while_rebuilding(self):
         Use Cases:
             Verifying that a pool can be destroyed during rebuild.
 
-        :avocado: tags=all,daily_regression,medium,pool,destroypoolrebuild
+        :avocado: tags=all,daily_regression,medium
+        :avocado: tags=pool,destroypoolrebuild
         """
         # Get the test parameters
         self.pool = TestPool(self.context, self.get_dmg_command())
         self.pool.get_params(self)
-        targets = self.params.get("targets", "/run/server_config/*")
-        rank = self.params.get("rank_to_kill", "/run/testparams/*")
-
-        # Create a pool
-        self.pool.create()
-
-        # Verify the pool information before starting rebuild
-        checks = {
-            "pi_nnodes": len(self.hostlist_servers),
-            "pi_ntargets": len(self.hostlist_servers) * targets,
-            "pi_ndisabled": 0,
-        }
-        self.assertTrue(
-            self.pool.check_pool_info(**checks),
-            "Invalid pool information detected prior to rebuild")
-
-        # Start rebuild
-        self.server_managers[0].stop_ranks([rank], self.d_log)
-        self.pool.wait_for_rebuild(True)
-
-        # Destroy the pool while rebuild is active
-        self.pool.destroy()
-
-        # Confirm the rebuild completes
-        self.pool.wait_for_rebuild(False)
-        self.log.info("Test Passed")
+        targets = self.params.get("targets", "/run/server_config/servers/*")
+        ranks = self.params.get("rank_to_kill", "/run/testparams/*")
```

Comment:
s/ranks/rank if we are using muxing for rank

Reply:
Keeping ranks as not changing the test to muxing.